pyproject.toml

```diff
@@ -47,10 +48,6 @@ Chat = "https://gitter.im/pybind/Lobby"
 [tool.setuptools]
 zip-safe = false
 
-[tool.setuptools.dynamic]
-license = "BSD"
-license-files = ["LICEN[CS]E*", "COPYING*", "NOTICE*", "AUTHORS*"]
```

Comment:
These are the default values already considered by `setuptools` and `wheel`.

Since it was clarified in the Python discourse that specifying `project.license` statically does not prevent the backend to dynamically populate `License-files` in the core metadata, having a dynamic `license` is no longer necessary.